pdns/dnsdist-lua.cc

```diff
@@ -237,10 +238,14 @@ void setupLuaConfig(bool client)
 			        sourceAddr = ComboAddress(source.substr(0, pos));
 			        sourceItf = itfIdx;
 			      }
+#ifdef SO_BINDTODEVICE
+                              /* we need to retain CAP_NET_RAW to be able to set SO_BINDTODEVICE in the health checks */
+                              g_capabilitiesToRetain.insert("CAP_NET_RAW");
```

Comment:
Indenting wrong or a github render error?

Reply:
I'll check but this function has a mix of indenting styles so it's always a pain. The upcoming reformat will be very welcome here!